File: apps/ios/Barns/BarnsApp/Domain/Entities/CompanyInfo.swift
import Foundation

struct CompanyInfo: Equatable, Identifiable, Sendable {
    let id: String
    let displayName: String
    let description: String
    let inquiryPolicy: String
    let phoneLabel: String
    let phoneNumber: String?
    let address: String?
    let businessHoursNote: String?
    let imageUrl: URL?
}

File: apps/android/app/src/main/java/com/barns/app/domain/model/CompanyInfo.kt
package com.barns.app.domain.model

data class CompanyInfo(
    val id: String,
    val displayName: String,
    val description: String,
    val inquiryPolicy: String,
    val phoneLabel: String,
    val phoneNumber: String?,
    val address: String?,
    val businessHoursNote: String?,
    val imageUrl: String?,
)
